src/ts/vttutils.ts

```diff
@@ -146,8 +207,11 @@ export namespace VttUtils {
     const vtt = cueContainer.vtt;
     const cueContainerDom = cueContainer.getDomElement();
 
+
     setDefaultVttStyles(cueContainerDom, vtt);
-    setVttWritingDirection(cueContainerDom, vtt, subtitleOverlaySize);
+
+    lineCount = countLines(cueContainer.getText());
+    setVttWritingDirectionAndCueBoxPositioning(cueContainerDom, vtt, subtitleOverlaySize);
```

Comment:
possibly to separate this to `setWritingDirection()` and `setCueBoxPositioning()`